public/docs/_examples/intro/dart/lib/hero_service.dart

```diff
@@ -0,0 +1,20 @@
+library developer_guide_intro.hero_service;
+
+import 'package:angular2/angular2.dart';
+import 'package:developer_guide_intro/hero.dart';
+import 'package:developer_guide_intro/backend_service.dart';
+import 'package:developer_guide_intro/logger_service.dart';
+
+@Injectable()
+class HeroService {
+  var _backendService;
+  var _logger;
+  HeroService(){
+    _logger = new Logger();
+    _backendService = new BackendService();
+  }
+  getHeroes(){
+    List<Hero> heroes = _backendService.getAll(Hero);
```

Comment:
Oh ignore that last comment. The TS version is using the logger service here to log the length of the heroes returned form backend service. I must have left that part out on accident.